src/main/resources/templates/fragments/ltft-summary/v1.0.0.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <title>LTFT summary</title>
+</head>
+<body>
+<table style="width:100%" border="1" th:fragment="ltftSummary(programme, startDate, cctDate, currentWte, newWte)">
+  <tr>
+    <td>Programme</td>
```

Comment:
Just a bit of minor thought.. 
Would the `<th>` instead of `<td>` makes  the table headers bold, so it will increase the readability?